privacyidea/lib/user.py

```diff
@@ -305,6 +305,8 @@ def info(self):
             # An empty user has no info
             return {}
         (uid, _rtype, _resolver) = self.get_user_identifiers()
+        if uid = None:
```

Comment:
We are close. 
Very close.
Please fix the failing tests by fixing the comparison:

~~~~python
if uid == None:
~~~~
Thanks!

Reply:
Done, thanks.